plugins/slick.resizer.js

```diff
@@ -85,8 +85,11 @@
       rightPadding: 0
     };
 
+    function setOptions(_newOptions){
+      options = $.extend(true, {}, _defaults, _newOptions);
+    }
     function init(grid) {
```

Comment:
nitpick can you add empty line between the 2 functions for a better separation

Reply:
think now it should look better